event-adapter/k8s-stackdriver-adapter.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="GO_MODULE" version="4">
```

Comment:
Please exclude this file - this file is relevant only to a particular IDE you're working with.